pom.xml

```diff
@@ -16,7 +16,7 @@
         <mvn.release.version>2.5.3</mvn.release.version>
 
         <junit.version>4.11</junit.version>
-        <consul-api.version>1.1.8</consul-api.version>
+        <consul-api.version>1.2.0.REBUY</consul-api.version>
```

Comment:
Do we want to jump to a new minor version? What if the next official release is a `1.1.9`? Then ours seems newer.